docs/data/whats-new/WhatsNewLayout.js

```diff
@@ -12,6 +12,39 @@ import Typography from '@mui/material/Typography';
 import { alpha } from '@mui/material/styles';
 
 const blogs = [
+  {
+    title: 'MUI X v6.18.0',
```

Comment:
his should be v6.18.1
```suggestion
    title: 'MUI X v6.18.1',
```

Reply:
Would something like this work better?

```suggestion
    title: 'MUI X v6.18.x',
```